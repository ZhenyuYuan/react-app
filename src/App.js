import React, { Component } from 'react';
import logo from './logo.svg';
import {Button} from 'antd';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <a
            className="App-link"
            href="https://reactjs.org"
            target="_blank"
            rel="noopener noreferrer"
          >
            Learn React
          </a>
          <Button>点我</Button>
        </header>
      </div>
    );
  }
}

export default App;
