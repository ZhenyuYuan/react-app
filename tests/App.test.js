import React from 'react';
import ReactDOM from 'react-dom';
import App from '../src/App';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});
// const div = document.createElement('div');
const component = renderer.create(
  <App />,
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
