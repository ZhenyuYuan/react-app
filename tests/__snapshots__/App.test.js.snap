// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
    <button
      className="ant-btn"
      onClick={[Function]}
      type="button"
    >
      <span>
        点 我
      </span>
    </button>
    <button
      className="ant-btn"
      onClick={[Function]}
      type="button"
    >
      <span>
        再点我
      </span>
    </button>
  </header>
</div>
`;
